### DISCUSSION TITLE
how can I set a root user, to see special section like Security Settings?

### DISCUSSION THREAD

**Discussion by arturoTellez** [NationalSecurityAgency/skills-service#2325] [General]
I try to make some setting, but my user is only admin

**Comment by sudo-may** [comment]
Hi @arturoTellez, root users can only be added by another root user. Please note that after skilltree dashboard is installed the very first user is assigned root role and then capable of adding other users as needed. To add a root user navigate to the dashboard's settings page (click on the user icon on the top-right then select `Settings` from the drop-down). On the Settings page click on the `Security` tab under the Navigation (security tab is only visible to users with the root role). You can now add root users within `Root Users Management` section. Hope this helps!

**Reply by arturoTellez** [reply]
Thanks, I changed it directily in the database. I don´t remenber see ". Please note that after skilltree dashboard is installed the very first user is assigned root role and then capable of adding other users as needed" in te documentation. It setting is cool.
